### DISCUSSION TITLE
Idea: Run a Markdown linter over the output

### DISCUSSION THREAD

**Discussion by pamelafox** [kjaymiller/cookiecutter-relecloud#175] [Ideas]
I found a Ruby package but the Python package that I found was real old. Be nice to check README.md

**Comment by kjaymiller** [comment]
What was the python package? I wonder if it's something that we could get up to date and use or if there's some guidance for basic linting similar to the [VS Code Extension](https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint).

Also it looks like markdownlint is available via pre-commit. This could be something that gets added to the repos.
